### PR TITLE
Add fifth exercise

### PR DESCRIPTION
**Pull request by odidukh**

### CODE REVIEW

File: ex05/ex05.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    for i in $@
+    do
+      (( sum += $i ))
+    done
+    echo "$sum";;
+  -e)
+    for i in $@
+    do
+      if !(($i % 2)); then
+      (( sum += $i ))
```
Comment (ashanaakh) [inline ex05/ex05.sh]: Wrong indentation here

**Comment by Oles225** [comment]
ex05

File: ex05/ex05.sh [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      fi
+    done
+    echo "$sum";;
+  -o)
+    for i in $@
+    do
+      if (($i % 2)); then
+      (( sum += $i ))
```
Comment (ashanaakh) [inline ex05/ex05.sh]: Wrong indentation here

File: ex05/ex05.sh [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+cnt=0;
+
+flag=$1
+
+shift
+
+re='^-?[0-9]+([.][0-9]+)?$'
+for i in $@; do
+if ! [[ $i =~ $re ]] ; then
+   echo "Error..";  exit 1
```
Comment (ashanaakh) [inline ex05/ex05.sh]: Remove one extra space after `echo "Error..";`

File: ex05/ex05.sh [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+fi
+done
+
+case $flag in
+  -s)
+    for i in $@
+    do
```
Comment (ashanaakh) [inline ex05/ex05.sh]: Better to put `do` in the same line with `for`

File: ex05/ex05.sh [hunk 5]
```diff
@@ -25,21 +25,20 @@ case $flag in
...
     do
       if !(($i % 2)); then
-      (( sum += $i ))
+        (( sum += $i ))
       fi
     done
     echo "$sum";;
   -o)
     for i in $@
     do
```
Comment (ashanaakh) [inline ex05/ex05.sh]: Still not fixed

**Pull request closed without merging**